src/core/ee/ee_jittrans.cpp

```diff
@@ -40,6 +42,30 @@ IR::Block EE_JitTranslator::translate(EmotionEngine &ee)
         if (branch_op)
             branch_delayslot = true;
 
+        //The EE kernel has a bug where it tries to disable interrupts, but due to pipelining this doesn't happen.
+        //The code in the kernel looks like this:
+        //* di
+        //* mfc0 k0, Status
+        //* ori k0, 0x13
+        //* mtc0 k0, Status
+        //* sync
+        //* eret
+        //DI is non-atomic, so MFC0 will read the *old* value of the Status register.
+        //The DI completes before MTC0 executes, so MTC0 will overwrite the old value of EIE.
+        //Some games (Namco 50th Anniversary, Jak X) check if interrupts are enabled during bootup and error
```

Comment:
Incorrect listing of games, the list is the folowing: Jak X, Namco 50th anniversary, Spongebob the Movie, Spongebob Battle for Bikini Bottom, The Incredibles, The Incredibles rize of the underminer, Soukou kihei armodyne, Garfield Saving Arlene, Tales of Fandom Vol. 2.